### PR TITLE
list-of-todos

### PR DESCRIPTION
**Pull request by DenisGurskiy**
https://DenisGurskiy.github.io/react_dynamic-list-of-todos/

### CODE REVIEW

File: src/components/TodoList/TodoList.tsx [hunk 1]
```diff
@@ -1,100 +1,151 @@
...
+            <p className="has-text-success">quo adipisci enim quam ut ab</p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button data-cy="selectButton" className="button" type="button">
+              <span className="icon">
+                <i className="far fa-eye" />
+              </span>
+            </button>
+          </td>
+        </tr> */}
```
Comment (anastasiiavorobiova) [inline src/components/TodoList/TodoList.tsx:147]: Consider removing unused lines of code

File: src/components/TodoModal/TodoModal.tsx [hunk 2]
```diff
@@ -1,49 +1,78 @@
...
+}) => {
+  const [user, setUser] = useState<User>();
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+
+    if (selectTodo?.userId) {
+      getUser(selectTodo?.userId)
+        .then(setUser)
```
Comment (anastasiiavorobiova) [inline src/components/TodoModal/TodoModal.tsx:24]: Consider adding a catch block here

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: src/components/TodoList/TodoList.tsx [hunk 3]
```diff
@@ -1,100 +1,86 @@
...
-  </table>
-);
+            <td className="is-vcentered is-expanded">
+              <p
+                className={classNames({
+                  'has-text-danger': !todo.completed,
+                  'has-text-success': todo.completed,
+                })}
+              >
+                {todo.title}
```
Comment (loralevitska) [inline src/components/TodoList/TodoList.tsx:60]: Destructure `todo`

**Review by loralevitska — APPROVED** [review]
Well done)

**Pull request closed without merging**